pkg/pg/onefieldresult.go

```diff
@@ -80,17 +85,20 @@ func (ofr Result) String() (s string) {
 	return fmt.Sprintf("{ %s }", strings.Join(results, ", "))
 }
 
+// Columns is a function that creates a column of all the results
```

Comment:
// Columns is a function that converts a Result into a list of strings.
// For every column the string value is added into the list.